Code/Tools/Libs/GuiFoundation/ContainerWindow/ContainerWindow.cpp

```diff
@@ -91,14 +91,6 @@ ezQtContainerWindow::~ezQtContainerWindow()
   ezQtDocumentWindow::s_Events.RemoveEventHandler(ezMakeDelegate(&ezQtContainerWindow::DocumentWindowEventHandler, this));
   ezToolsProject::s_Events.RemoveEventHandler(ezMakeDelegate(&ezQtContainerWindow::ProjectEventHandler, this));
   ezQtUiServices::s_Events.RemoveEventHandler(ezMakeDelegate(&ezQtContainerWindow::UIServicesEventHandler, this));
-
-  // The dock manager does not take ownership of dock widgets.
-  auto dockWidgets = m_DockManager->dockWidgetsMap();
```

Comment:
@Sanakan8472 I suspect that this code is no longer neccessary. Do you remember what the issue back then was when you wrote this code?

Reply:
@Ingrater as the comment states, the dock manager does not take ownership of the dock widgets so they would just leak. You can verify this by just putting a breakpoint in one of the dtors. Make sure to test both shown and hidden docks widgets.

Reply:
Seems to be working fine now. maybe the behaviour did change when we updated ADS the last time?

Reply:
Yes, probably